### ISSUE TITLE
Timeout error fetching content from Squarespace

### ISSUE THREAD

**Issue by patbegg** [NodeSquarespace/node-squarespace-server#162]
Hi,
I've got a problem with the node server hanging when it's fetching data from SS. In the log i'll get the following in the log:

```
> sqs-log Squarespace server running localhost:5050
> sqs-log Logging into Squarespace...
> sqs-log ...Logged in to Squarespace
> sqs-log Fetching data from Squarespace...
> sqs-log Fetched fullData for collection 56414d8ee4b061de63ab6a0b
> sqs-log Fetched fullData for collection 56414d86e4b0de34bd8ed5af
> sqs-log Fetched fullData for collection 56414d52e4b0de34bd8ed225
...
> sqs-log Fetched fullData for collection 5691f76027c801a804a35e2a
> sqs-log Fetched fullData for collection 56923819841aba578bf72297
> sqs-log Fetched fullData for collection 5691f75f27c801a804a35e1a
> sqs-log Logging into Squarespace...
> sqs-log ...Logged in to Squarespace
> sqs-log Fetching data from Squarespace...
```

It looks like it logs back in again and fetches more content. This works the first time but the second or third time it just hangs.

Any ideas?

**Comment by kitajchuk** [comment]
Hmm. This is weird, haven't seen it behave this way yet.

**Comment by kitajchuk** [comment]
Can you hit me with any more information? Like, template and so forth?

**Comment by patbegg** [comment]
I have managed to solve this in a hacky way. I tracked the problem down to x2 specific pieces of content it was trying to sync (x2 imported wordpress blog posts). There was nothing obviously wrong with the content e.g. embedded video or large image, but after deleting the problem items and deleting local cache it all seemed to sync correctly. It was the Base Template being used.

I did take a look at obvious problems with the items so I could report back for others with problems but there was nothing obvious other than being imported.
